tests/test_parallel.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+
+from datasets.parallel import parallel_backend, ParallelBackendConfig
+from datasets.utils.py_utils import map_nested
+
+from .utils import require_joblibspark
+
+
+def add_one(i):  # picklable for multiprocessing
+    return i + 1
+
+
+@require_joblibspark
+def test_parallel_backend_input():
+    with parallel_backend("spark", steps=["downloading"]):
```

Comment:
I think you can replace "downloading" by "download". Adding -ing at the end of every step name is not necessary imo :)

Reply:
+1